testing_and_setup/compass/ocean/jigsaw_to_MPAS/mesh_definition_tools.py

```diff
@@ -198,97 +218,31 @@ def AtlanticPacificGrid(lat, lon, cellWidthInAtlantic, cellWidthInPacific):
                     cellWidthOut[j, i] = cellWidthInAtlantic[j]
     return cellWidthOut
 
-#
-# def  circleOnGrid(lon, lat, centerLon, centerLat, radius, tanhWidth)
```

Comment:
@mark-petersen, is it okay to get rid if this commented-out function?

Reply:
Yes. That one and `lldistkm` are the last of the matlab helper functions I wrote two years ago, the first time we interfaced with the jigsaw front end (matlab at that time). It's nice to see how far we've come since then!